tests/ptf_runner.py

```diff
@@ -1,17 +1,21 @@
 import pipes
 import traceback
 import logging
+from datetime import datetime
 
 logger = logging.getLogger(__name__)
 
 
 def ptf_collect(host, log_file):
-    host.fetch(src=log_file, dest='./logs/ptf_collect/', flat=True, fail_on_missing=False)
     pos = log_file.rfind('.')
-    pcap_file = (log_file[0:pos] if pos > -1 else log_file) + '.pcap'
+    filename_prefix = log_file[0:pos] if pos > -1 else log_file
+
```

Comment:
For this change, it seems to carry the folder name as well, which is not reasonable.
My original opinion is if log file looks like "`/tmp/somecase-logfile.log`", then just use `somecase-logfile` as prefix.
If log file has multiple folders, looks like this `/tmp/subfolder/somecase-logfile.log`, then just use `somecase-logfile.log` as prefix too.

Reply:
Yeah, make sense